src/plugins/irc-events/link.js

```diff
@@ -16,10 +16,9 @@ module.exports = function(irc, network) {
 		}
 
 		var links = [];
-		var split = data.message.split(" ");
+		var split = data.message.replace(/\x02|\x1D|\x1F|\x16|\x0F|\x03(?:[0-9]{1,2}(?:,[0-9]{1,2})?)?/g, "").split(" ");
```

Comment:
For the record/archive purpose, this is the same check as [this one](https://github.com/thelounge/lounge/blob/05ec819f2d74fb2e2d13dc135e169ddd21852143/client/js/lounge.js#L779), to clean up messages from colors, boldness, ...